Tools/Scripts/webkitpy/port/mac.py

```diff
@@ -225,6 +225,11 @@ def operating_system(self):
     def default_child_processes(self, **kwargs):
         default_count = super(MacPort, self).default_child_processes()
 
+        # FIXME: arm64 Mac hardware can handle more processes than the default number we calculate.
```

Comment:
Intel Macs were already implicitly doubling due to hyper threading (counting 2 logical cores for each physical core). That might explain why explicitly doubling for arm64 Macs (which don't have hyper-threading) roughly works out.

Given the .75 multiplier below, we're essentially running all bots now at 1.5X over-capacity relative to physical CPUs.

Longer term, when we look at more sophisticated scheduling, it might be best to have our Intel bots use their _physical_ CPU count as the input to any math we do, so that the math works out across architectures.